pennylane/qnn/cost.py

```diff
@@ -15,8 +15,18 @@
 This submodule contains frequently used loss and cost functions.
 """
 # pylint: disable=too-many-arguments
+from warnings import warn
+
 import pennylane as qml
 
+WARNING_STRING = (
+    "SquaredErrorLoss will no longer be directly imported in PennyLane from "
+    "release 0.16.0. It will be accessible by importing the qnn "
+    "module:\n\nimport pennylane as qml\nimport pennylane.qnn\n\n"
+    "qml.qnn.SquaredErrorLoss\n\nConsider adding 'import pennylane.qnn' "
```

Comment:
```suggestion
    "module:\n\nimport pennylane as qml\nfrom pennylane import qnn\n\n"
    "qnn.SquaredErrorLoss\n\nConsider using 'from pennylane import qnn' "
```

Reply:
Thanks! I went for a simple version:
> SquaredErrorLoss will no longer be directly imported in PennyLane from release 0.16.0. It will be accessible by importing the qnn module. Consider adding 'from pennylane import qnn' to your existing code now.

(same for the other two instances below)